scenes/performance/ingame_user_score_viewer.gd

```diff
@@ -0,0 +1,35 @@
+extends PanelContainer
+
+@onready var uname_label = %uname_label
+@onready var uscore_label = %uscore_label
+
+
+var stats : IngameUser :
```

Comment:
There should be no spaces before the colons on this line, so var stats: IngameUser: